### PR TITLE
add dynamic list and gen details

### LINKED ISSUE #523: Show dynamic generation details
child of #520

### PR DESCRIPTION
**Pull request by artem-rozhkov-dev-optima**
close #523 #521

### CODE REVIEW

File: components/loadMoreButton/index.vue [hunk 1]
```diff
@@ -4,6 +4,7 @@
       face="round"
       fill="neutral"
       extend
+      @click="loadMore()"
     >
       Load More
     </AeButton>
```
Comment (emilsomm) [inline components/loadMoreButton/index.vue]: this needs to be wrapped in a `<no-ssr>aebutton</no-ssr>` this should already be in there. i must have forgot to do this. That should solve the problem for now. also change the import to 
change this to 
`````
let AeButton

if (process.browser) {
  AeButton = require('@aeternity/aepp-components-3').AeButton
}
`````

File: components/loadMoreButton/index.vue [hunk 2]
```diff
@@ -16,6 +17,11 @@ export default {
   name: 'LoadMoreButton',
   components: {
     AeButton
+  },
+  methods: {
+    loadMore () {
+      this.$emit('update')
```
Comment (emilsomm) [inline components/loadMoreButton/index.vue]: ❤️

File: pages/generations/index.vue [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
+        :key="number"
+        :to="`/generations/${generation.keyBlock.height}`"
+        class="generation-link"
+      >
+        <Generation
+          :data="generation"
+        />
+      </nuxt-link>
     </Generations>
+    <LoadMoreButton @update="loadMoreGen" />
```
Comment (emilsomm) [inline pages/generations/index.vue]: 😍

File: components/loadMoreButton/index.vue (hunk 1 above)
Comment (artem-rozhkov-dev-optima) [reply components/loadMoreButton/index.vue]: updated

**Pull request merged**